easypy/concurrency.py

```diff
@@ -343,15 +360,16 @@ def logged_wait(self, timeout=None, initial_log_interval=None):
 
 
 def _run_with_exception_logging(func, args, kwargs, ctx):
+    ctx.update(threadname=threading.current_thread().name)
```

Comment:
```python
ctx['threadname'] = threading.current_thread().name
```

(yea, I know it's unrefactored code, but there is no reason not to use the conventional style here...)